chart/compass/charts/connectivity-adapter/templates/virtualservice-mtls.yaml

```diff
@@ -25,6 +25,7 @@ spec:
             {{- range .Values.global.gateway.headers.request.remove }}
             - {{ . }}
             {{- end }}
+      timeout: {{ .Values.global.oathkeeper.timeout }}
```

Comment:
how about adding a destination rule for oathkeeper so that the timeout can be centrally defined on it ?